drivers/sensor/sensirion/sen6x/shell.c

```diff
@@ -0,0 +1,579 @@
+#define SEN6X_STOP_HELP         SHELL_HELP("Stop continuous measurement", "<device>")
+#define SEN6X_RESET_HELP        SHELL_HELP("Reset device", "<device>")
+#define SEN6X_PRODUCT_NAME_HELP SHELL_HELP("Get product name", "<device>")
+#define SEN6X_SERIAL_HELP       SHELL_HELP("Get serial number", "<device>")
+#define SEN6X_FIRMWARE_HELP     SHELL_HELP("Get firmware version", "<device>")
+#define SEN6X_SET_TEMP_OFFSET_HELP                                                                 \
+	SHELL_HELP("Set temperature offset parameters",                                            \
+		   "<device> <offset> <slope> <time_constant> <slot>")
+#define SEN6X_SET_TEMP_ACCEL_HELP                                                                  \
+	SHELL_HELP("Set temperature acceleration parameters", "<device> <K> <P> <T1> <T2>")
+#define SEN6X_SET_VOC_TUNING_HELP                                                                  \
+	SHELL_HELP(                                                                                \
+		"Set VOC algorithm tuning parameters",                                             \
+		"<device> <index_offset> <learning_time_offset_hours> <learning_time_gain_hours> " \
+		"<gating_max_duration_minutes> <std_initial> <gain_factor>")
+#define SEN6X_SET_NOX_TUNING_HELP                                                                  \
+	SHELL_HELP(                                                                                \
+		"Set NOx algorithm tuning parameters",                                             \
+		"<device> <index_offset> <learning_time_offset_hours> <learning_time_gain_hours> " \
+		"<gating_max_duration_minutes> <std_initial> <gain_factor>")
+#define SEN6X_SET_CO2_CALIB_HELP   SHELL_HELP("Enable CO2 automatic self calibration", "<on|off>")
+#define SEN6X_SET_PRESSURE_HELP    SHELL_HELP("Set ambient pressure", "<ambient_pressure>")
+#define SEN6X_SET_ALTITUDE_HELP    SHELL_HELP("Set sensor altitude", "<sensor_altitude>")
+#define SEN6X_CLEAN_FAN_HELP       SHELL_HELP("Start fan cleaning", "<device>")
+#define SEN6X_ACTIVATE_HEATER_HELP SHELL_HELP("Activate SHT heater", "<device>")
+#define SEN6X_VOC_STATE_SAVE_HELP                                                                  \
+	SHELL_HELP("Save VOC algorithm state in a global variable", "<device>")
+#define SEN6X_VOC_STATE_RESTORE_HELP                                                               \
+	SHELL_HELP("Restore VOC algorithm state from a global variable", "<device>")
+
+static long sen6x_get_long(const struct shell *sh, const char *arg, int *out_ret, const char *name)
+{
+	int ret = 0;
+	long value = shell_strtol(arg, 0, &ret);
+
+	if (ret) {
+		shell_error(sh, "Invalid %s: %s", name, arg);
+		*out_ret = ret;
+	}
+
+	return value;
+}
+
+static unsigned long long sen6x_get_ull(const struct shell *sh, const char *arg, int *out_ret,
+					const char *name)
+{
+	int ret = 0;
+	unsigned long long value = shell_strtoull(arg, 0, &ret);
+
+	if (ret) {
```

Comment:
https://docs.zephyrproject.org/latest/contribute/coding_guidelines/index.html#misrac-rule-14-4

There're more entries I've preferred no to mark to avoid cluttering